src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -989,6 +989,35 @@ public static ListContainersParam containersCreatedBefore(final String id) {
     public static ListContainersParam withContainerSizes(final Boolean size) {
       return create("size", String.valueOf(size));
     }
+    
+    /**
+     * Show containers with a label value
+     *
+     * @return ListContainersParam
+     */
+    public static ListContainersParam withLabel(String key, String value) {
+      try {
+        return create("filters", 
+          URLEncoder.encode("{\"label\":[\"" + key + '=' + value + "\"]}", "UTF-8"));
+      } catch (UnsupportedEncodingException e) {
+        // Should never happen
+        throw Throwables.propagate(e);
+      }
+    }
+    
+    /**
+     * Show containers with a label
+     *
+     * @return ListContainersParam
+     */
+    public static ListContainersParam withLabel(String key) {
+      try {
+        return create("filters", URLEncoder.encode("{\"label\":[\"" + key + "\"]}", "UTF-8"));
```

Comment:
seems like this could just call `withLabel(key, null)` and then the other version could check if value is null or not